### PR TITLE
feat: event explore

### PR DESCRIPTION
**Pull request by tyyzqmf**
----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

## Summary

(describe what this merge request does)

## Implementation highlights

截屏2023-08-29 17 26 58

## Test checklist

- [ ] add new test cases
- [ ] all code changes are covered by unit tests
- [ ] end-to-end tests
  - [ ] deploy web console with CloudFront + S3 + API gateway
  - [ ] deploy web console within VPC
  - [ ] deploy ingestion server
    - [ ] with MSK sink
    - [ ] with KDS sink
    - [ ] with S3 sink
  - [ ] deploy data processing
  - [ ] deploy data modeling
    - [ ] new Redshift Serverless
    - [ ] provisioned Redshift
    - [ ] Athena
  - [ ] deploy with reporting

## Is it a breaking change

- [ ] add parameters without default value in stack
- [ ] introduce new service permission in stack
- [ ] introduce new top level stack module

## Miscellaneous

- [ ] introduce new symbol link source file(s) to be shared among infra code, web console frontend, and web console backend

### CODE REVIEW

File: frontend/src/components/layouts/Navigation.tsx [hunk 1]
```diff
@@ -60,11 +60,11 @@ const Navigation: React.FC<INavigationProps> = (props: INavigationProps) => {
...
   const navAnalyticsItems: SideNavigationProps.Item[] = [
-    {
-      type: 'link',
-      text: t('nav.analytics.realtime'),
-      href: `/analytics/${projectId}/app/${appId}/realtime`,
-    },
+    // {
+    //   type: 'link',
+    //   text: t('nav.analytics.realtime'),
+    //   href: `/analytics/${projectId}/app/${appId}/realtime`,
```
Comment (zxkane) [inline frontend/src/components/layouts/Navigation.tsx:66]: Will enable those links?

**Review by zxkane — CHANGES_REQUESTED** [review]

File: frontend/src/components/layouts/Navigation.tsx (hunk 1 above)
Comment (tyyzqmf) [reply frontend/src/components/layouts/Navigation.tsx:66]: wait pipeline realtime

File: frontend/src/pages/analytics/event/AnalyticsEvent.tsx [hunk 2]
```diff
@@ -96,290 +145,628 @@ const AnalyticsEvent: React.FC = () => {
...
+                  <Header
+                    variant="h2"
+                    actions={
+                      <SpaceBetween direction="horizontal" size="xs">
+                        <Button
+                          iconName="refresh"
+                          onClick={resetConfig}
+                          loading={loadingData}
+                        >
+                          {t('analytics:button.reset')}
```
Comment (chenhaiyun) [inline frontend/src/pages/analytics/event/AnalyticsEvent.tsx:457]: move to: `common.button.reset`

File: frontend/src/pages/analytics/event/AnalyticsEvent.tsx [hunk 3]
```diff
@@ -96,290 +145,628 @@ const AnalyticsEvent: React.FC = () => {
...
+                        </Button>
+                        <Button
+                          variant="primary"
+                          loading={loadingData}
+                          onClick={() => {
+                            setSelectDashboardModalVisible(true);
+                          }}
+                        >
+                          {t('analytics:button.save')}
```
Comment (chenhaiyun) [inline frontend/src/pages/analytics/event/AnalyticsEvent.tsx:466]: move to `common.button.save`

File: frontend/src/pages/analytics/event/AnalyticsEvent.tsx [hunk 4]
```diff
@@ -96,290 +145,628 @@ const AnalyticsEvent: React.FC = () => {
...
+                  </SpaceBetween>
+                </ColumnLayout>
+                <br />
+                <Button
+                  variant="primary"
+                  iconName="search"
+                  onClick={clickPreview}
+                  loading={loadingData}
+                >
+                  {t('common:button.preview')}
```
Comment (chenhaiyun) [inline frontend/src/pages/analytics/event/AnalyticsEvent.tsx:716]: common is default. we can use `button.preview` directly

**Review by zxkane — APPROVED** [review]

**Pull request merged**